internal/postgresql/utils.go

```diff
@@ -68,8 +68,23 @@ func setPassword(ctx context.Context, connParams ConnParams, username, password
 	}
 	defer db.Close()
 
-	_, err = dbExec(ctx, db, fmt.Sprintf(`alter role "%s" with password '%s';`, username, password))
-	return err
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+
+	query := fmt.Sprintf("set local log_statement = %s", pq.QuoteLiteral("none"))
+	if _, err = tx.ExecContext(ctx, query); err != nil {
+		err = tx.Rollback()
```

Comment:
assigning tx.Rollback to error will override the real error. I'll just ignore the Rollback error.

Reply:
@sgotti That's true. I'm not so familiar with Go, but I agree with you. I updated the PR already.